lib/horizon/operation/bump_footprint_expiration.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Stellar.Horizon.Operation.BumpFootprintExpiration do
+  @moduledoc """
+  Represents a `BumpFootprintExpiration` operation from Horizon API.
+  """
+
+  @behaviour Stellar.Horizon.Resource
+
+  alias Stellar.Horizon.Mapping
+
+  @type t :: %__MODULE__{
+          ledgers_to_expire: String.t()
+        }
```

Comment:
- We could write the typespec in one line.
- `ledgers_to_expire` should be a non-negative integer instead of a string, no? In the tests, you have it as an integer. Please check how Horizon returned this value and update the code/tests properly.